webhook.go

```diff
@@ -108,7 +110,15 @@ func main() {
 				if matchLoadedHook(hook.ID) != nil {
 					log.Fatalf("error: hook with the id %s has already been loaded!\nplease check your hooks file for duplicate hooks ids!\n", hook.ID)
 				}
-				log.Printf("\tloaded: %s\n", hook.ID)
+
+				msg := fmt.Sprintf("\tloaded: %s", hook.ID)
+
+				// initialize concurrency map
+				if hook.MaxConcurrency > 0 {
```

Comment:
You need to add this logic to the `reloadHooks` method as well.

When the webhook reloads hooks following things can happen:
- new hook has been added
- existing hook was modified
- existing hook was removed 

So you should reflect those changes to the `limits` map

Reply:
fixed